src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts

```diff
@@ -12,7 +12,7 @@ import {
 import {
   DEFAULT_PAYLOAD_MAX_VALUE,
   DEFAULT_PAYLOAD_MIN_VALUE,
-} from 'modules/Builder/components/ConditionRow/ConditionRow.const';
+} from 'modules/Builder/components/ConditionRow_old/ConditionRow.const';
```

Comment:
Also, in this file and the next one: if we only import constants from the old component and turn the flag 'on', will it work? Are the constants unchanged in the new component, making them equivalent? 
Or just the import path need to be replaced in deleting old files process?

Reply:
The same as for previous, I will revert it to `ConditionRow` since there are no differences.